### DISCUSSION TITLE
disaster-of-house/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#129] [Giscus]
# disaster-of-house/

2024 年 4 月 16 日早上，作者接到妈妈电话说家被台风吹跑了。作者回家后发现房顶被吹走，决定自己掏钱盖新房。村里共有 8 户受灾，但只是厢房的房顶被吹走。重建过程中遇到天气不好的情况，最终在村民帮助下完成了重建。期间姑父为帮忙而杀鸡、鸭，并做了三锅菜。整个经历让作者感觉像是梦境一样，同时也意识到生活中的困难和挑战并没有停止。

https://www.eallion.com/disaster-of-house/

**Comment by lmm214** [comment]
人没事就好，要不就原拆原造，来个别墅吧～

**Reply by eallion** [reply]
谢谢！
现在地不好批，自建房面积和楼层也很多限制。主要是现在也不回去住，等老了再说。

**Comment by rebron1900** [comment]
太危险了，虽然我没面对过台风，但是我们这之前也有一次大风天气，一些小区楼顶的违建彩钢棚都能直接被吹走。
结合你拍的照片，似乎全是木质结构。
这种结构的房子没有被完全吹垮已是万幸。

**Reply by eallion** [reply]
谢谢！
是的，这次也吹走了一些楼顶的瓦棚。其实那种老的瓦片反而没事，就是这种大块大块的瓦容易吹走。

**Comment by wangyunzi** [comment]
农村就是这样的，但是只要人都是好好的就行了。

**Reply by eallion** [reply]
谢谢！
还是房屋不够结实。
